build/publish.settings.xml

```diff
@@ -1,16 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
-  <servers>
-    <server>
-      <id>ossrh</id>
-      <username>${env.OSSRH_USR}</username>
-      <password>${env.OSSRH_PSW}</password>
-    </server>
-    <!--  This entry can be removed for projects built on public travis (travis-ci.com) -->
-    <server>
-      <id>na-artifactory</id>
-      <username>${env.ARTIFACTORY_USER}</username>
-      <password>${env.ARTIFACTORY_APIKEY}</password>
-    </server>
-  </servers>
+    <servers>
+        <server>
+            <id>ossrh</id>
+            <username>${env.OSSRH_USR}</username>
+            <password>${env.OSSRH_PSW}</password>
+        </server>
+        <server>
+            <id>na-artifactory</id>
+            <username>${env.ARTIFACTORY_USER}</username>
+            <password>${env.ARTIFACTORY_APIKEY}</password>
+        </server>
+        <!-- when this settings file is used, central is overridden by artifactory, so need those creds here -->
+        <server>
```

Comment:
These are needed for running the `test` target under the `staging-and-test` profile. Given that this file is no longer just related to publishing, I wonder if we should rename it to `jenkins.settings.xml`?